randovania/games/super_metroid/json_data/Crateria.txt

```diff
@@ -629,7 +638,8 @@ Extra - asset_id: 17
   * Layers: default
   * Pickup 7; Category? Major
   > Door to Flyway
-      Trivial
+      # TODO: proper torizo requirements
+      Normal Damage ≥ 30
 
```

Comment:
In the vanilla game you're expected to fight BT with nothing. I think its fair that Normal Damage >= 99 and then (Normal Damage >= 30 AND Beginner Combat).

I also think its fine as it is. The boss really isn't complicated or difficult.

Reply:
> In the vanilla game you're expected to fight BT with nothing

Technically wrong. You're expected to have 99 energy and between 5/10 missiles.

> I think it's fair that Normal Damage >= 99 and then (Normal Damage >= 30 AND Beginner Combat).

This will not do what you think it does. All damage numbers are multipled by 1.5 in the starter preset, so this would be equivalent to 149 damage (more than you can get in vanilla trickless) or 45 + beginner combat.

I think the following is nice though:
- items:
- 5 missiles or beginner combat
- Health:
- 65 damage or (beginner combat + 30 damage) or (intermediate combat)

Reply:
I was not aware of the way health was multiplied throughout presets.